target/arc/semfunc-v3.c

```diff
@@ -14580,6 +14580,97 @@ arc_gen_DMPYWHU(DisasCtxt *ctx, TCGv a, TCGv b, TCGv c)
     return DISAS_NEXT;
 }
 
+
+int
+arc_gen_VMIN2(DisasCtxt *ctx, TCGv a, TCGv b, TCGv c)
```

Comment:
Please add a comment section describing what this function implements. You can use PRM for this. The same for VMAX2